src/util/system.cpp

```diff
@@ -1284,10 +1284,13 @@ std::string CopyrightHolders(const std::string& strPrefix)
     const auto copyright_devs = strprintf(_(COPYRIGHT_HOLDERS).translated, COPYRIGHT_HOLDERS_SUBSTITUTION);
     std::string strCopyrightHolders = strPrefix + copyright_devs;
 
-    // Make sure Bitcoin Core copyright is not removed by accident // Sugarchain: Branding // Do not change
+    /*
+    // Make sure Bitcoin Core copyright is not removed by accident
     if (copyright_devs.find("Bitcoin Core") == std::string::npos) {
         strCopyrightHolders += "\n" + strPrefix + "The Bitcoin Core developers";
```

Comment:
code is duplicated. put here the copyright holders.

Reply:
![image](https://user-images.githubusercontent.com/60179867/98434258-26085780-209c-11eb-8255-e03bce39cedf.png)

Reply:
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/60179867/98434322-944d1a00-209c-11eb-83ef-ab6fb7db9ed1.png)

Reply:
style is different. so we keep both separate style.